client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -491,13 +491,19 @@ class EmailProvidersComparison extends React.Component {
 	}
 
 	handleBack = () => {
-		const { selectedSite } = this.props;
+		const { selectedSite, handleBack } = this.props;
 
-		page( emailManagement( selectedSite.slug ) );
+		page( handleBack ?? emailManagement( selectedSite ) );
```

Comment:
Was the change from `emailManagement( selectedSite.slug )` to `emailManagement( selectedSite )` intentional? I think the previous value was correct.
```suggestion
		page( handleBack ?? emailManagement( selectedSite.slug ) );
```